src/appSelector.ts

```diff
@@ -33,6 +34,11 @@ export interface BuildUseCase {
   name: string;
   options: string;
 }
+export interface Variant {
```

Comment:
Maybe call this `VariantList` Or `Variants` ? Or at least call the variable in the `App` interface `variants` ? It seems confusing to only have a singular `variant` when it contains an array of values...

Reply:
Ok for naming the type `VariantList`, and the variable `variants`.